hbase-server/src/main/java/org/apache/hadoop/hbase/rsgroup/RSGroupInfoManagerImpl.java

```diff
@@ -1244,4 +1244,11 @@ public synchronized void renameRSGroup(String oldName, String newName) throws IO
     setRSGroup(updateTables, newName);
   }
 
+  @Override
+  public synchronized void updateRSGroupConfig(String groupName, Map<String, String> configuration)
+      throws IOException {
+    RSGroupInfo rsGroupInfo = getRSGroupInfo(groupName);
```

Comment:
Add one log here? Update from xxx to xxx?

Reply:
Already loged in MaterRpcServices.updateRSGroupConfig like this

```
LOG.info("{} update rsgroup {} configuration {}", master.getClientIdAuditPrefix(), groupName,
        configuration);
```